funnel/templates/password_login_form.html.jinja2

```diff
@@ -1,7 +1,7 @@
 {% from "baseframe/forms.html.jinja2" import renderfield, rendersubmit %}
 
-{% macro passwordlogin(loginform, markup) %}
-  <form id="form-passwordlogin" method="POST" class="mui-form mui-form--margins" accept-charset="UTF-8">
+{% macro passwordlogin(loginform, ref_id, markup) %}
```

Comment:
Drop the markup parameter. It's used for the forgot password link, but that link should not be in field helper text as it can't be translated. It has to move out of the form.